build/images/scripts/start_ovs

```diff
@@ -144,6 +149,16 @@ set -euo pipefail
 # exit with code 128 + SIGNAL
 trap "quit" INT TERM
 
+if [ "$skip_kmod" == "true" ]; then
+    # ovs-ctl start will invoke ovs-kmod-ctl to load the openvswitch Kernel module if necessary
+    # (using modprobe). In some cases, this can fail unexpectedly, for example, with Talos Linux
+    # (see https://github.com/antrea-io/antrea/issues/5707). This is why this script offers the
+    # skip-kmod flag, which prevents the ovs-ctl script from trying to load any Kernel module. In
+    # order for this to work, we need to turn ovs-kmod-ctl into a "no-op".
+    cp /usr/share/openvswitch/scripts/ovs-kmod-ctl /usr/share/openvswitch/scripts/ovs-kmod-ctl.bak
+    echo ":" > /usr/share/openvswitch/scripts/ovs-kmod-ctl
```

Comment:
Does `:` has any special meaning as a script?

Reply:
It should be no-different from an empty script. I used `:` to emphasize that it was a shell script and that we meant for it to be a no-op.